frontend/scss/_bootstrap-colors.scss

```diff
@@ -0,0 +1,8 @@
+﻿$primary: #0339E1;
```

Comment:
Would recommend to use the same layout as bootstrap and to copy the whole variables file since it makes maintaining the SCSS easier - https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss

Reply:
Since I expect to only need to override 10-20 variables, this would make the file cluttered in my opinion and I'd be afraid to be unable to see what has actually been overridden (out of 1000+ variables)

Reply:
Agree 👍